test/NuGet.Core.Tests/NuGet.ProjectModel.Test/ProjectLockFile/Packages.LockFileUtilities.IsLockFileStillValidTests.cs

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Commands;
+using NuGet.ProjectModel.Test.Builders;
+using Xunit;
+using static NuGet.Frameworks.FrameworkConstants;
+using PackagesLockFileBuilder = NuGet.ProjectModel.Test.Builders.PackagesLockFileBuilder;
+
+namespace NuGet.ProjectModel.Test.ProjectLockFile
+{
+    public partial class LockFileUtilities
```

Comment:
Why `partial`?

Reply:
I don't like 10,000 line files and have seen this pattern of using partial classes for tests to group related and split unrelated tests into different files, while keeping a consistent namespace, class name, method name convention. Utility classes, like the one being tested by this class, tend to have methods that are completely unrelated, from a testing point of view, so I'm trying to set up a good structure.

Reply:
Makes sense.  For now let's defer this new use of `partial` until it's warranted.  Given the current size distribution of files it does not make sense to introduce a new pattern that addresses the <1% case.  `partial` adds no value here and now but can be added later as necessary.